packages/expo-updates/utils/src/vcs.ts

```diff
@@ -10,7 +10,7 @@ export interface Client {
 }
 
 export default async function getVCSClientAsync(projectDir: string): Promise<Client> {
-  if (await isGitInstalledAsync()) {
+  if (await isGitInstalledAndConfiguredAsync()) {
```

Comment:
this seems like something we should extract to a package that we share between eas-cli and expo-updates cli, and potentially other places where we want to do this same thing (or library authors in the community want to)

not a blocker for this pr but something to think about in the future

Reply:
Agreed.